### DISCUSSION TITLE
Access x-data value outside of its scope

### DISCUSSION THREAD

**Discussion by anjanesh** [alpinejs/alpine#3820] [1. Help]
```html
<div x-data="{ showCountries: false }">
	<div>
	    <label class="relative inline-flex items-center cursor-pointer">
	        <input
	            type="checkbox"
	            id="optCountries"
	            @click="showCountries = !showCountries"
	            class="sr-only peer"
	        />
	        <div class="so-many-class-names"></div>
	        <span class="cls1">Show Countries</span>
	    </label>
	</div>
	<div>
		<ul x-show="showCountries">
			<li>India</li>
			...
		</ul>
	</div>
</div>
```
`document.getElementById('optCountries').checked = false;`
I need to set `showCountries` to false outside of its scope.

How can I achieve this ?
Is `Alpine.store` the only way out ?

**Comment by ekwoka** [comment]
multiple ways.

Store is probably the BEST way.

but you could also do `Alpine.$data(element).showCountries = false`

Or put `@disable-countries.window="showCountries = false"` on the component and from wherever just do `window.dispatchEvent(new Event('disable-countries'))`

**Reply by anjanesh** [reply]
I've switched to using Store but find the lines a bit long.
```
@click="$store.local_data.aws.showCountries = !$store.local_data.aws.showCountries"
```

**Reply by ekwoka** [reply]
Did you try just having `aws` as the store?

or even just `showCountries`?

you can also put methods on Stores to just have a toggle on it

```js
$store.showCountries.toggle()
```

**Reply by anjanesh** [reply]
I agree thats easier - but I am having it structured. There's actually a lot of data.
